Code/Editor/LyViewPaneNames.h

```diff
@@ -41,9 +41,9 @@ namespace LyViewPane
     static const char* const SubstanceEditor = "Substance Editor";
     static const char* const LandscapeCanvas = "Landscape Canvas";
     static const char* const AnimationEditor = "EMotion FX Animation Editor";
-    static const char* const PhysXConfigurationEditor = "PhysX Configuration (PREVIEW)";
+    static const char* const PhysXConfigurationEditor = "PhysX Configuration";
 
-    static const char* const SliceRelationships = "Slice Relationship View (PREVIEW)";
+    static const char* const SliceRelationships = "Slice Relationship View";
```

Comment:
I seem to remember that these strings were also used to store the window settings for the tools, among other things. Have we double checked that changing this will not break something?
(I think there was a way to pass both a new and legacy name to circumvent this)